api/v1beta1/common_types.go

```diff
@@ -181,3 +182,12 @@ func SetupDefaults() {
 
 	SetupManilaDefaults(manilaDefaults)
 }
+
+// GetEnvDefault is a function that gets the value associated with key from environment
+// variables, but use baseDefault as a value in the case of an empty string
+func GetEnvDefault(key string, baseDefault string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return baseDefault
+}
```

Comment:
this is already in lib-common https://github.com/openstack-k8s-operators/lib-common/blob/696d321a942e9617c3049d3f1ae0944f20b9ce58/modules/common/util/env.go#LL22C1-L22C1